src/libProtoServer/ServerResponse.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto2";
+
+package ZilliqaMessage;
+
+import "ZilliqaMessage.proto";
+import "ServerMessages.proto";
+
+
+message DefaultResponse
+{
+    required string result = 1 [default = "hello"];
+}
+
+message UIntResponse
+{
+    required uint32 result = 1;
+}
+
+message StringResponse
+{
+    required string result = 1;
+}
+
+message DoubleResponse
+{
+    required double result = 1 [default = 0];
+}
+
+
+
+message CreateTransactionResponse
+{
+    optional string error = 1;
+    optional string info = 2;
+    optional string tranid = 3;
+    optional string contractaddress = 4;
+}
+
+message GetTransactionResponse
+{
+    optional string error = 1;
+    optional ProtoTransaction tx = 2;
+    optional string receipt = 3;
+}
+
+message GetDSBlockResponse
+{
+    optional string error = 1;
+    optional ProtoDSBlock dsblock = 2;
+}
+
+message GetTxBlockResponse
+{
+    optional string error = 1;
+    optional ProtoTxBlock txblock = 2;
+}
+
+message GetBalanceResponse
+{
+    optional string error = 1;
+    optional ByteArray balance = 2;
```

Comment:
String instead of ByteArray, as discussed

Reply:
Okay